lib/Config.php

```diff
@@ -36,25 +35,6 @@ public function __construct($configDir) {
                 $this->readData($config);
             }
         }
-
-        $this->debug($this->getValue('info', 'debug', false));
-    }
-
-    public function debug($set = null) {
-        if (is_null($set)) {
-            return $this->debugMode;
-        }
-        elseif ($set) {
-            ini_set('display_errors', 1);
-            ini_set('error_reporting', E_ALL);
-            $this->debugMode = true;
-        }
-        else
-        {
-            ini_set('display_errors', 0);
-            ini_set('error_reporting', E_ALL & ~E_NOTICE & ~E_STRICT & ~E_DEPRECATED);
-            $this->debugMode = false;
-        }
```

Comment:
The problem I can see here is that we can't set the server's debug config anymore. I think we should keep the `ini_set()`s maybe based on the `log_level` in the config and set it in `Mesamatrix::init()` along with the other PHP ini config like the timezone.

Or maybe the new logger handles already that?